Raindrops/main.js

```diff
@@ -0,0 +1,299 @@
+  if (z === '/' && x % y !== 0 ) {
+    x = y * Math.floor(Math.random()* 10); 
+    return x + z + y;
+  } else if (x >= y) {
+    return x + z + y; 
+  } else {
+    return y + z + x;
+  }
+}
+
+function createDrop() {
+  const drop = document.createElement('div');
+  drop.classList.add('rain-drop');
+  let equation = createEquation(1, 10);
+  if (score < 10) {
+    equation;
+  } else if (score >= 10 && score <= 40) {
+    equation = createEquation(1, 15);
+  } else {
+    equation = createEquation(1, 25);
+  }
+  drop.setAttribute('data-value', new Function(`return ${equation}`)());
+  mainRainDrop.appendChild(drop);
+  drop.style.left = Math.random() * 60 + '%';
+  drop.append(equation);
+}
+
+function createBonusDrop() {
+  const bonusDrop = document.createElement('div');
+  bonusDrop.classList.add('bonus-drop');
+  let equation = createEquation(1, 10);
+  bonusDrop.setAttribute('data-value', new Function(`return ${equation}`)());
+  mainRainDrop.appendChild(bonusDrop);
+  bonusDrop.style.left = Math.random() * 60 + '%';
+  bonusDrop.append(equation);
+}
+
+function removeDrop() {
+  const rainDrop = document.querySelector('.rain-drop');
+  mainRainDrop.removeChild(rainDrop);
+}
+
+function removeBonusDrop() {
+  const bonusDrop = document.querySelector('.bonus-drop');
+  mainRainDrop.removeChild(bonusDrop);
+}
+
+function clearScreenAfterClickEnterAndRemoveDrop() {
+  const rainDrop = document.querySelector('.rain-drop');
+  const bonusDrop = document.querySelector('.bonus-drop');
+  if (rainDrop && rainDrop.getAttribute('data-value') === screen.value ) {
+  rainDrop.classList.add('explosion');
+  songWin.play();
+  setTimeout(removeDrop,100);
+  score++;
+  scoreTable.textContent = score;
+  screen.value = '';
+  rainDrop.textContent = '';
+  } else if (bonusDrop && bonusDrop.getAttribute('data-value') === screen.value) {
+    deleteAllWhenBonusDrop();
+    songWin.play();
+    score += 3;
+    scoreTable.textContent = score;
+    screen.value = '';
+    bonusDrop.textContent = '';
+  } else if ((bonusDrop || rainDrop) && (rainDrop.getAttribute('data-value') !== screen.value || bonusDrop.getAttribute('data-value') !== screen.value)) {
+    songError.play();
+    score--;
+    scoreTable.textContent = score;
+    screen.value = '';
+  }
+}
+
+function removeDropWhenDropTouchWave(){
+  let heightOfWave;
+  const waveSizes = wave.getBoundingClientRect();
+  const topOfWave = Math.floor(waveSizes.top);
+
+  const rainDrop = document.querySelector('.rain-drop');
+  const rainDropSizes = rainDrop ? rainDrop.getBoundingClientRect(): 0;
+  const bottomOfDrop = Math.floor(rainDropSizes.bottom);
+
+  const bonusRainDrop = document.querySelector('.bonus-drop');
+  const bonusRainDropSizes = bonusRainDrop ? bonusRainDrop.getBoundingClientRect(): 0;
+  const bottomOfBonusRainDrop = Math.floor(bonusRainDropSizes.bottom) 
+
+  if (dropTouchWave === 0 && (bottomOfDrop >= topOfWave || bottomOfBonusRainDrop >= topOfWave)) {
```

Comment:
Could we move `bottomOfDrop >= topOfWave || bottomOfBonusRainDrop >= topOfWave` to a variable as we use it in each condition?